packages/kuma-gui/src/main.ts

```diff
@@ -48,11 +47,10 @@ async function mountVueApplication() {
           ...$,
           ...msw.TOKENS,
           ...fakeFs.TOKENS,
-          ...x.TOKENS,
         }
         return [
           ...application.services(TOKENS),
-          ...xdebug.services(TOKENS),
+          ...serviceMeshDebug.services(TOKENS),
```

Comment:
This was just because I moved the `x` debug container to be in `service-mesh` instead.